client-sdk/go/testing/testing.go

```diff
@@ -15,9 +15,10 @@ import (
 
 // TestKey is a key used for testing.
 type TestKey struct {
-	Signer  signature.Signer
-	Address types.Address
-	SigSpec types.SignatureAddressSpec
+	SecretKey [32]byte
```

Comment:
Are all secret keys the same length? A slice would probably make more sense. Also you don't seem to set this for ed25519 test keys. You can get it by using something like:
```golang
ms := memorySigner.NewTestSigner(seed)
sk := ms.(signature.UnsafeSigner).UnsafeBytes()
```

Reply:
Also @matevz added a similar thing in #878.

Reply:
I'll use Matevz's one